tests/test_crosstalk.py

```diff
@@ -91,28 +90,47 @@ def construct(imageList):
             image.getArray()[:] += self.noise
             return image
 
+        # Construct detector
+        detName = 'detector'
+        detId = 1
+        detSerial = 'serial'
+        orientation = cameraGeom.Orientation()
+        pixelSize = lsst.geom.Extent2D(1, 1)
+        bbox = lsst.geom.Box2I(lsst.geom.Point2I(0, 0),
+                               lsst.geom.Extent2I(2*width, 2*height))
+        crosstalk = np.array(self.crosstalk, dtype=np.float32)
+
+        camBuilder = cameraGeom.Camera.Builder("fakeCam")
+        detBuilder = camBuilder.add(detName, detId)
+        detBuilder.setSerial(detSerial)
+        detBuilder.setBBox(bbox)
+        detBuilder.setOrientation(orientation)
+        detBuilder.setPixelSize(pixelSize)
+        detBuilder.setCrosstalk(crosstalk)
+
+        # ccd = lsst.afw.cameraGeom.Detector("detector", 123, lsst.afw.cameraGeom.SCIENCE, "serial",
+        #                                    amplifiers, lsst.afw.cameraGeom.Orientation(),
+        #                                    lsst.geom.Extent2D(1, 1), {},
+        #                                    np.array(self.crosstalk, dtype=np.float32))
```

Comment:
Don't leave commented-out code: remove the comment characters or remove the code.